pkg/graveler/graveler_test.go

```diff
@@ -1239,14 +1271,16 @@ func testGravelerCommit(t *testing.T, kvEnabled bool) {
 				return
 			}
 			expectedAppliedData := testutil.AppliedData{
-				Values:      values,
+				Values:      tt.values,
 				MetaRangeID: expectedRangeID,
 			}
 			if tt.args.sourceMetarange != nil {
 				expectedAppliedData = testutil.AppliedData{}
 			}
-			if diff := deep.Equal(tt.fields.CommittedManager.AppliedData, expectedAppliedData); diff != nil {
-				t.Errorf("unexpected apply data %s", diff)
+			if kvEnabled || tt.name != "valid commit with staging and sealed" { // TODO - delete when removing the DB
```

Comment:
It looks like we are running this test for 'db' but no compare the results. Prefer to use t.Skip and specify the case and not the name of the test.
For example looks like when !kvEnabled && tt.values != values we would like to skip.

Reply:
done